core/templates/pages/exploration-player-page/layout-directives/content-language-selector.component.ts

```diff
@@ -30,9 +30,6 @@ import { PlayerPositionService } from
   'pages/exploration-player-page/services/player-position.service';
 import { PlayerTranscriptService } from
   'pages/exploration-player-page/services/player-transcript.service';
-import { SwitchContentLanguageRefreshRequiredModalComponent } from
```

Comment:
If this is the only place this component is being used, and we've now removed it, can we get rid of the component from the codebase?

Reply:
@nithinrdy Definitely!

I have checked for the component coverage and it is only used here to refresh the page when the user inputs something and selects the language change option.

Here's a video showing the changes when the component is completely removed from the codebase.

https://user-images.githubusercontent.com/96513964/198983250-f97d3065-2732-46e5-80db-f3bb7551b27d.mp4